src/Markdown.MAML/Renderer/MamlRenderer.cs

```diff
@@ -75,331 +31,161 @@ public string MamlModelToString(IEnumerable<MamlCommand> mamlCommands)
+        private static IEnumerable<XElement> GenerateParagraphs(string text)
         {
-            PushTag("command:syntax");
-            foreach (MamlSyntax syntaxItem in command.Syntax)
+            if (text != null)
             {
-                PushTag("command:syntaxItem");
-                _stringBuilder.AppendFormat("<maml:name>{0}</maml:name>{1}", XmlEscape(command.Name), Environment.NewLine);
-                foreach (MamlParameter parameter in syntaxItem.Parameters)
-                {
-                    AddParameter(command, parameter, inSyntax: true);
-                }
-                PopTag("command:syntaxItem");
+                return text.Split(new string[] { Environment.NewLine }, StringSplitOptions.None)
+                    .Select(para => new XElement(mamlNS + "para", para));
             }
-            PopTag("command:syntax");
+
+            return Enumerable.Empty<XElement>();
         }
 
-        private void AddLinks(MamlCommand command)
+        private static XElement CreateSyntaxItem(MamlSyntax syntaxItem, MamlCommand command)
         {
-            PushTag("command:relatedLinks");
-            foreach (MamlLink Link in command.Links)
-            {
-                if (Link.IsSimplifiedTextLink)
-                {
-                    // that should never happen.
-                    // we don't know how to represent non-parsed links in maml.
-                    // skipping it.
-                    // TODO: error out?
-                    continue;
-                }
-
-                PushTag("maml:navigationLink");
-
-                PushTag("maml:linkText");
-                _stringBuilder.Append(XmlEscape(Link.LinkName));
-                PopTag(1);
-
-                PushTag("maml:uri");
-
-                // PowerShell help engine is not happy, when LinkUri doesn't represent a valid URI
-                // but it's often convinient to have it (i.e. relative links between markdown files on github).
-                // so, we are ignoring non-uri links when rendering the final maml.
-                // https://github.com/PowerShell/platyPS/issues/164
-                Uri uri;
-                string linkUri = "";
-                try
-                {
-                    uri = new Uri(Link.LinkUri);
-                    linkUri = XmlEscape(Link.LinkUri);
-                }
-                catch (UriFormatException e)
-                {
-                }
-
-                _stringBuilder.Append(linkUri);
-                PopTag(1);
-
-                PopTag(1);
-            }
-            PopTag(1);
+            return new XElement(commandNS + "syntaxItem", 
+                    new XElement(mamlNS + "name", command.Name),
+                    syntaxItem.Parameters.Select(param => CreateParameter(param, true)));
         }
 
-        private void AddExamples(MamlCommand command)
+        private static XElement CreateParameter(MamlParameter param, bool syntax)
```

Comment:
nit: please use `isSyntax` or `inSyntax` for this `bool` param name: it's good to be bale to tell that's something is a bool by name.